colcon-test/.lcovrc

```diff
@@ -0,0 +1,163 @@
+#
```

Comment:
Also, could you show me the defaults values of each parameter, and add some words to the dictionary?